example/lib/app.dart

```diff
@@ -55,7 +58,23 @@ class _LMSampleAppState extends State<LMSampleApp> {
                   if (ConnectionState.done == snapshot.connectionState &&
                       snapshot.hasData &&
                       snapshot.data!.success) {
-                    return const LMFeedScreen();
+                    return FutureBuilder(
```

Comment:
we have now three future builders nested in each other, is'nt it a bad practice?
can we do somethind like this :
 ValidateUserResponse? validateUser;
 MemberStateResponse? memberState;
 GetCommunityConfigurationsResponse? communityConfigurations;
// can use a better name here:)
Future<void> setUpUser()async {
validateUser=await LMFeedCore.instance.validateUser();
memberState = await LMFeedCore.instance.getMemberState();
communityConfigurations =await LMFeedCore.instance.getCommunityConfigurations();
}

and then assign this async function to future of FutureBuider

Reply:
Sure, there is a function in Future class, Future.wait([]) it takes a list of future argumkents and wait for there response. We just need to check if all the calls are successful or not as they are a blocker for us

Reply:
Fixed in the coming PR, we can resolved this conversation for now.

Reply:
https://github.com/LikeMindsCommunity/likeminds-feed-flutter/pull/43